fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/DerbyBootstrapper.java

```diff
@@ -111,7 +113,7 @@ public static void bootstrap(Connection connection, String adminSchemaName, Stri
         // Current version history for the database. This is used by applyWithHistory
         // to determine which updates to apply and to record the new changes as they
         // are applied
-        VersionHistoryService vhs = new VersionHistoryService(adminSchemaName, dataSchemaName, OAUTH_SCHEMANAME);
+        VersionHistoryService vhs = new VersionHistoryService(adminSchemaName, dataSchemaName, OAUTH_SCHEMANAME, BATCH_SCHEMANAME);
```

Comment:
cool so, we support or are going to support upgrading of OAUTH and BATCH tables.

Reply:
Absolutely.